lib/fog/ovirt/requests/compute/attach_volume.rb

```diff
@@ -0,0 +1,22 @@
+module Fog
+  module Compute
+    class Ovirt
+      class Real
+        def attach_volume(id, options)
+          raise ArgumentError, "instance id is a required parameter" unless id
+          raise ArgumentError, "volume id is a required parameter for attach-volume" unless options.key? :id
+
+          client.attach_volume(id, options[:id])
```

Comment:
Should I also revert back and remove braces around `attrs` from `Fog::Compute::Ovirt::Server#attach_volume` and `Fog::Compute::Ovirt::Server#detach_volume` methods definition ? Personally, I would keep the braces.